Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:6
+
+RUN npm install -g \
+      bower \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends \
+      nginx \
+ && rm -rf /var/lib/apt/lists/* \
+ && rm /var/www/html/index.nginx-debian.html \
+ && ln -sf /dev/stdout /var/log/nginx/access.log \
+ && ln -sf /dev/stderr /var/log/nginx/error.log
+
+COPY . /usr/src/app
+
+WORKDIR /usr/src/app
+
+RUN npm install \
+ && bower install --allow-root \
+ && npm run-script build -- --environment=production \
```

Comment:
`npm run build -- --environment=production` should be fine.. No need for `npm run-script`, nobody uses this anymore.

Reply:
Also in 0dcb81aaa2cccd0d4f9a94fbf4ee7d877d5d2cd4